Task1/Task1/MatrixMultiplication.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1
+{
+    /// <summary>
+    /// Class with functions of matrix multiplication.
+    /// </summary>
+    public class MatrixMultiplicationClass
+    {
+        /// <summary>
+        /// Multiplicate the matrix.
+        /// </summary>
+        public static int[,] Multiplication(int[,] a, int[,] b)
+        {
+            var timeStart = DateTime.Now;
+            ArgumentNullException.ThrowIfNull(a);
+            ArgumentNullException.ThrowIfNull(b);
+            if (a.Length / a.GetLength(0) != b.GetLength(0))
+            {
+                throw new Exception("Матрицы нельзя перемножить");
+            }
+            int[,] newMatrix = new int[a.GetLength(0), b.GetLength(1)];
```

Comment:
```suggestion
            var newMatrix = new int[a.GetLength(0), b.GetLength(1)];
```
Не пишите тип в объявлении дважды.